src/snowflake/snowpark/_internal/ast_utils.py

```diff
@@ -1342,12 +1342,31 @@ def ClearTempTables(message: proto.Request) -> None:
 
 def base64_str_to_request(base64_str: str) -> proto.Request:
     message = proto.Request()
-    message.ParseFromString(base64.b64decode(base64_str.strip()))
+    message.ParseFromString(base64.b64decode(base64_str))
     return message
 
 
-def base64_str_to_textproto(base64_str: str) -> str:
-    request = base64_str_to_request(base64_str)
+def merge_requests(messages: List[proto.Request]) -> proto.Request:
+    message = proto.Request()
```

Comment:
nit: maybe add docstring comment on what the function does (i.e., merge all requests into single request. Order is the same as messages).

nit: rename potentially messages -> requests?

Reply:
Empty case is also not handled in this function. I.e.

```
if not messages:
   return message
```

Reply:
Updated